src/constants.rs

```diff
@@ -80,5 +81,8 @@ pub const ACTOR_DEF_ANON: ActorDef = ActorDef::anonymous();
 pub const ACTOR_DEF_MASTER_TOKEN: ActorDef = ActorDef::system("master-token", "admin");
 pub const ACTOR_DEF_TESTBED: ActorDef = ActorDef::system("testbed", "testbed");
 
+pub const SCHEDULER_INTERVAL_SECONDS_REPUBLISH: u32 = 600;
```

Comment:
If I read correctly this 600 value was previously coded as 120 directly in `scheduler.rs`. Is this an intentional increase and does it relate to #370 or #423 in some way? Is it perhaps yet another way to reduce the amount of stored data by not performing the republish unnecesarily often or something?